code/solutions/troubleshooting-provisioning-errors/example_sqs_queues.yaml

```diff
@@ -0,0 +1,42 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
Please rename the file to `sqs-queue.yaml`

Reply:
@rezabekf thank you - will use `sqs-queues.yaml` as the intent is to have more than one queue in this lab example.